pkg/ovs/ovsctl/ofctl.go

```diff
@@ -176,7 +171,7 @@ func (c *ovsCtlClient) runOfctlCmd(openflow15 bool, cmd string, args ...string)
 	return out, nil
 }
 
-func (c *ovsCtlClient) RunOfctlCmd(cmd string, args ...string) ([]byte, error) {
+func (c *ovsCtlClient) RunOVSCtlCmd(cmd string, args ...string) ([]byte, error) {
 	// Default to use Openflow15.
 	return c.runOfctlCmd(true, cmd, args...)
```

Comment:
the method is called `RunOVSCtlCmd` but calls `runOfctlCmd`? That doesn't really make sense. `ovs-appctl`, `ovs-ctl`, `ovs-ofctl` are all different things.